.github/workflows/plugin_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Plugin TasksTimetable Tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  actions: read
+  checks: none
+  contents: read
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: none
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  PluginTests:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: run tests
+        uses: matomo-org/github-action-tests@v1
+        with:
+          git-head: ${{ github.head_ref }}
+          git-sha: ${{ github.event.pull_request.head.sha }}
+          plugin-name: 'TasksTimetable'
+          php-version: '7.4'
```

Comment:
I think we used to run UI tests on the minimum supported PHP version only, as that helps to find compatibility issues with code not covered by other tests. So might be good to change that to `7.2.` again.

Reply:
change to 7.2